vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -13,7 +13,11 @@ Entando includes the Swagger UI in a quickstart environment and is reachable at
 
 The Swagger UI can be enabled or disabled in a running container by modifying the SPRING_PROFILES_ACTIVE environment variable for the entando-de-app container. 
 
-1. Edit the deployment. The name may be different outside of a quickstart environment.
+1. Edit the deployment. The name may be different outside of a quickstart environment. Note: The host machine can communicate with Kubernetes directly using the [ent CLI](../getting-started/entando-cli.md) and `ent kubectl`. Use `sudo kubectl` when sending a command from within the VM.
```

Comment:
the note is confusing

how does the host machine communicate with K8s directly when using the ent CLI? Since ent is playing mediator, how is that direct? Maybe take out directly

the subject is ent  and then in the last sentence, use sudo kubectl is really confusing because that is completely unexpected. Maybe start the sentence with When sending a command from... so the reader knows the frame of reference has changed. This really threw me off